src/reconcile/rec_visibility.c

```diff
@@ -488,48 +464,14 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
 
             /* Find the update this tombstone applies to. */
             if (!__wt_txn_upd_visible_all(session, upd)) {
-                upd = __get_valid_upd(upd, tombstone, &same_txn_valid_upd);
+                while (upd->next != NULL && upd->next->txnid == WT_TXN_ABORTED)
```

Comment:
Tip- maybe write this as:
`for (; upd->next != NULL && upd->next->txnid == WT_TXN_ABORTED; upd = upd->next;)`

Ignore this if you like it the way it's written.

Reply:
As there is no performance optimization with the suggested change, I feel the existing change may be more readable. I fine to change if others also feel the same.